src/backend/joanie/settings.py

```diff
@@ -309,6 +311,15 @@ class Development(Base):
 class Test(Base):
     """Test environment settings"""
 
+    JOANIE_LMS_BACKENDS = [
```

Comment:
As discussed the other day, maybe don't put it here but as a settings override on the the TestCase where you need it?

Reply:
Oh yes ! You are right!